realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.java

```diff
@@ -68,17 +68,14 @@ public void generate() throws IOException, UnsupportedOperationException {
         imports.add("io.realm.internal.TableOrView");
         imports.add("io.realm.internal.ImplicitTransaction");
         imports.add("io.realm.internal.LinkView");
-        imports.add("io.realm.internal.android.JsonUtils");
         imports.add("java.io.IOException");
         imports.add("java.util.ArrayList");
         imports.add("java.util.Collections");
         imports.add("java.util.List");
-        imports.add("java.util.Date");
```

Comment:
I didn't find any direct usage of the `Date` class. If I missed something, I'll obviously add it back.